openlibrary/core/ia.py

```diff
@@ -117,10 +117,10 @@ def edition_from_item_metadata(itemid, metadata):
 def get_cover_url(item_id):
     """Gets the URL of the archive.org item's title (or cover) page."""
     base_url = f'{IA_BASE_URL}/download/{item_id}/page/'
-    title_response = requests.head(base_url + 'title.jpg', allow_redirects=True)
-    if title_response.status_code == 404:
-        return base_url + 'cover.jpg'
-    return base_url + 'title.jpg'
+    cover_response = requests.head(base_url + 'cover.jpg', allow_redirects=True)
```

Comment:
I noticed this last review, but do we have any data to show which one is better, cover vs. title?

I have a feeling we might be alternating between two less than perfect options.

I  _thought_ title.jpg was always better because if it existed it was either the title page of a book with a plain (e.g. cloth) cover, or an image of the front cover with a clear title. If title.jpg didn't exist, it was more than likely the cover was a modern book so cover.jpg was good, or in the worst case we'd get a blank cover image because there was nothing better available.

I'm not even sure all scanned items have a title.jpg, where as they do have a cover.jpg, even if it is a blank cloth of leather board.

Reply:
This is a good point I had not considered. I just noticed that when I was importing books it was using the title page and not the cover image, but I should have stopped to consider why that might be. I have reverted this change. :)

Reply:
Let's try this endpoint: #7616